options/posix/include/pwd.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 
 struct passwd {
 	char *pw_name;
+	char *pw_passwd;
```

Comment:
This is an ABI break.

Reply:
what is the plan for resolving that?

Reply:
First, we should make sure that the structs are 100% complete now (e.g., compared to Linux). Then we merge it then but also announce the break.